added OIP-18 - 2way OKChain payment system

nickchisiu opened the pull request.

nickchisiu committed:
added OIP-18 - 2way OKChain payment system

oceanfanfan reviewed (commented):
Hidden files should not be added here and OIP-18 is already used by others, please create your own one

nickchisiu committed:
removed hidden files and changed the OIP to valid one

nickchisiu commented:
> Hidden files should not be added here and OIP-18 is already used by others, please create your own one

Sure thing. I've removed the hidden files and changed the OIP to the valid one.

Thank you !
